locale/en/locale.cfg

```diff
@@ -318,7 +318,7 @@ centrifugal-pan=Huge place to concentrate and refine large amounts of materials.
 jig=Gravity concentrator for intermediate fluid-based ore processing.
 grease-table=Using grease to select the best ores and split from the rest using X-ray scans.
 regolite-mine=Dig and extract regolites from this planet.
-genlab=A place to create and maturate genetically altered organisms.
```

Comment:
as far as i can tell this isnt actually incorrect even if its not actually used much. py likes his fancy words

Reply:
Today I learned! Will open a new pull request, since I messed up the branches on this one.